### PR TITLE
feat : created a landing page

### PR DESCRIPTION
**Pull request by GeetaPogul**
Created a landing page of E-commerce in react

### CODE REVIEW

File: src/components/js/Home.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+        <div className='cards-container'>
+
+
+        <div class="card">
+ 
+           < img class="card-image" src="../images/imac/imac1.jpg" alt="" />
+
+             {/* <div class="top-icon-red ">
```
Comment (manojasarsa) [inline src/components/js/Home.js]: You can remove these comments if no longer needed.

File: src/components/js/Home.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ 
+           < img class="card-image" src="../images/ipad/ipad1.jpg" alt="" />
+
+             {/* <div class="top-icon-red "> 
+                  <i class="fas fa-heart card-heart-icon" > </i> 
+              </div> */}
+
+             <div class="btn-bar btn-macbook"> <a href="index.html" > <button class="card-btn btn1">   ipad  </button> </a> </div>
+        </div>
+
```
Comment (manojasarsa) [inline src/components/js/Home.js]: You can remove these extra spaces.

File: src/components/js/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                
+            </button> </a>     
+           </div>
+
+            
+        </div>
+
+        </div>
+    </nav>
+
```
Comment (manojasarsa) [inline src/components/js/Navbar.js]: Here as well. you can remove blank lines

**Review by manojasarsa — APPROVED** [review]
Rest looks good.

File: src/index.js [hunk 4]
```diff
@@ -2,16 +2,23 @@ import React from 'react';
...
+import Navbar from "./components/js/Navbar.js"
+import Home from "./components/js/Home.js"
+
 
 ReactDOM.render(
   <React.StrictMode>
+    <Navbar />
+    <Home />
     <App />
+
```
Comment (himadri2110) [inline src/index.js]: You should put the `Nav`, `Home`, and all other components inside the `App.js` component and not in `index.js`

File: src/components/js/Home.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+function Home() {
+  return (
+    <div>
+      
+      <div className='bg-image'>
+        <img className="home-image" src='../images/home-img.jpg' /> 
+
+        <a className='text-on-image' href='#'> <h1> Explore ! </h1> </a>
+       </div>
```
Comment (himadri2110) [inline src/components/js/Home.js]: You should use `Link` from react-router to navigate to other pages of your app. `a` tag has to be used only for external links.

**Review by himadri2110 — APPROVED** [review]
Apart from the suggestions, it looks good to go

**Pull request merged**